auto_rx/station.cfg.example

```diff
@@ -151,19 +160,46 @@ max_altitude = 50000
 # Discard positions more than 1000 km from the observation station location (if set)
 max_radius_km = 1000
 
-# MQTT (Even more interfacing options!)
-#   Post all sonde messages to a MQTT server
-[mqtt]
-mqtt_enabled = False
-mqtt_hostname = 127.0.0.1
-mqtt_port = 1883
+# Advanced Settings
+# These control low-level settings within various modules.
+# Playing with them may result in odd behaviour.
+[advanced]
+# Scanner - Receive bin width (Hz)
+search_step = 800
+# Scanner - Minimum SNR for a peak to be detected. The lower the number, the more peaks detected.
+snr_threshold = 10
+# Scanner - Maximum number of peaks to search through during a scan pass.
+#	Increase this if you have lots of spurious signals, though this will increase scan times.
+max_peaks = 10
+# Scanner - Minimum distance between peaks (Hz)
+min_distance = 1000
+# Scanner - Scan Dwell Time - How long to observe the specified spectrum for.
+scan_dwell_time = 20
+# Scanner - Detection Dwell time - How long to wait for a sonde detection on each peak.
+detect_dwell_time = 5
+# Scanner - Delay between scans. We should delay a short amount between scans to allow for decoders and other actions to jump in.
+scan_delay = 10
+# Quantize search results to x Hz steps. Useful as most sondes are on 10 kHz frequency steps. 
+quantization = 10000
+# Upload when (seconds_since_utc_epoch%upload_rate) == 0. Otherwise just delay upload_rate seconds between uploads.
+# Setting this to True with multple uploaders should give a higher chance of all uploaders uploading the same frame,
+# however the upload_rate should not be set too low, else there may be a chance of missing upload slots.
+synchronous_upload = True
+# Only accept a payload ID as valid until it has been seen N times.
+# This helps avoid corrupted callsigns getting onto the map.
+payload_id_valid = 5
+# Paths to the rtl_fm and rtl_power utilities. If these are on your system path, then you don't need to change these.
+sdr_fm_path = rtl_fm
+sdr_power_path = rtl_power
+
 
-# Rotator Settings
+# Rotator Settings - Currently not implemented.
```

Comment:
should we have rotator before we merge - does anyone use it?

Reply:
Not sure if there are any users of it at the moment. Don't think so. 
I'll get it re-added once I have my new Az/El system up, and can actually test the code.